tuf/keys.py

```diff
@@ -1016,8 +1053,66 @@ def format_rsakey_from_pem(pem):
   # Ensure the PEM string has a valid header and footer.  Although a simple
   # validation of 'pem' is performed here, a fully valid PEM string is
   # needed to later successfully verify signatures.
-  pem_header = '-----BEGIN PUBLIC KEY-----'
-  pem_footer = '-----END PUBLIC KEY-----'
+  public_pem = extract_pem(pem) 
+
+  # Begin building the RSA key dictionary. 
+  rsakey_dict = {}
+  keytype = 'rsa'
+  public = public_pem 
```

Comment:
Resolved [here in PR #311 with removal of variable](https://github.com/theupdateframework/tuf/pull/311/files#diff-a3be85d1a9209356a591a86fedcad4a5R1070).